src/QMCDrivers/Crowd.h

```diff
@@ -83,6 +85,9 @@ class Crowd
     estimator_manager_crowd_.accumulate(mcp_walkers_, walker_elecs_, walker_twfs_, walker_hamiltonians_, rng);
   }
 
+  /// Collect walker log data
+  void collect(int current_step);
```

Comment:
Please call it collectStepWalkerLog. We may adjust the name if we expand its use.

Reply:
Isn't it redundant to repeat the class name in a member function name?

Reply:
Seems redundant to include the class name in the name of one of its member functions.  Will do it.

Reply:
Sure

Reply:
Done